config-handler-router/pom.xml

```diff
@@ -24,6 +24,16 @@
     <packaging>bundle</packaging>
 
     <dependencies>
+        <!-- TODO: Uncomment shaded jar at version 2.10.0-->
```

Comment:
This comment and unused dependency should be removed. Then we should be able to merge.